examples/integration-scripts/challenge.test.ts

```diff
@@ -119,5 +120,16 @@ test('challenge', async () => {
 
     // Check Bob's challenge in conctats
     contacts1 = await client1.contacts().list();
-    expect(contacts1[0].challenges[0].authenticated).toEqual(true);
+    bobContact = findContact(contacts1, 'bob');
```

Comment:
```suggestion
    bobContact = contacts1.find(contact => contact.name === "bob");
```

Just a suggestion that would work equally well, but can avoid the extra function?